R/json_nest.R

```diff
@@ -0,0 +1,33 @@
+#' JSON nest
+#'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
+#' A wrapper around [tidyr::nest()] which stores the nested data into JSON columns.
+#'
+#' @param .data A data frame, a data frame extension (e.g. a tibble), or  a lazy data frame (e.g. from dbplyr or dtplyr).
+#' @param .names_sep If `NULL`, the default, the names will be left as is.
+#' @param ... <[`tidy-select`][tidyr_tidy_select]> Columns to pack, specified
+#'   using name-variable pairs of the form `new_col = c(col1, col2, col3)`.
+#'   The right hand side can be any valid tidy select expression.
+#' @seealso [tidyr::nest], [json_nest_join]
+#' @export
+#' @examples
+#' df <- tibble::tibble(x = c(1, 1, 1, 2, 2, 3), y = 1:6, z = 6:1)
+#' nested <- json_nest(df, data = c(y, z))
+#' nested
+json_nest <- function(.data, ..., .names_sep = NULL) {
+  UseMethod("json_nest")
+}
+
+#' @export
+json_nest.data.frame <- function(.data, ..., .names_sep = NULL) {
+  dot_nms <- ...names()
```

Comment:
`check_suggested()` here and elsewhere??

Reply:
oops! done in https://github.com/cynkra/dm/pull/918/commits/e3c35260c34270cb02ebd06b7fe17bcf302a8b52